<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimCav 5.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icons/logo-tg3.png</normaloff>Icons/logo-tg3.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_cavity">
       <attribute name="title">
        <string>Cavity</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_stability">
       <attribute name="title">
        <string>Stability</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="stabilityUI" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Varying element:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="stability_comboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Variation range:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="stability_xstart">
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="placeholderText">
              <string>(mm)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="stability_xend">
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="placeholderText">
              <string>(mm)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_calcStability">
             <property name="text">
              <string>Calculate!</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_beamSize">
       <attribute name="title">
        <string>Beam Size</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Beam size at element:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="beamsize_comboBox_watch"/>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Varying element:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="beamsize_comboBox_var"/>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Variation range:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="beamsize_xstart"/>
           </item>
           <item>
            <widget class="QLineEdit" name="beamsize_xend"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_calcBeamsize">
             <property name="text">
              <string>Calculate!</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_crossSection">
       <attribute name="title">
        <string>Cross Section</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>z (mm) = </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="crossSectionBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="crossSectionSlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_crossSectionUpdate">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>Icons/update.png</normaloff>Icons/update.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QScrollArea" name="scroll_cavityIcons">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scroll_cavityIconsArea">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>809</width>
         <height>48</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1145</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="view_Toolbar"/>
    <addaction name="view_editElements"/>
    <addaction name="view_calculateSolutions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionEdit"/>
   <addaction name="actionCalculator"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="elementbox">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>310</width>
     <height>240</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Cavity elements</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>7</x>
          <y>5</y>
          <width>281</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>#</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>Element</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>d or R</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>n or þ</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="button_moveUp">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/arrowUp.png</normaloff>Icons/arrowUp.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_moveDown">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/arrowDown.png</normaloff>Icons/arrowDown.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="button_delete">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Icons/small_del.png</normaloff>Icons/small_del.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QScrollArea" name="element_list">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QWidget" name="element_list_scroll">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>290</width>
          <height>415</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="button_calcCavity">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icons/cavity_go1.png</normaloff>Icons/cavity_go1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>220</width>
         <height>35</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="modifyCavity">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>310</width>
     <height>240</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>310</width>
     <height>240</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Modfy cavity</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="modifyCavity_widget">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>0</y>
       <width>288</width>
       <height>216</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="button_flatMirror">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/e_flat_mirror.png</normaloff>Icons/e_flat_mirror.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="button_curvedMirror">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/e_curved_mirror.png</normaloff>Icons/e_curved_mirror.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="button_distance">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/e_distance.png</normaloff>Icons/e_distance.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_block">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/e_block.png</normaloff>Icons/e_block.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="button_brewsterPlate">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/e_brewster_plate.png</normaloff>Icons/e_brewster_plate.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="button_brewsterCrystal">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/e_brewster_crystal.png</normaloff>Icons/e_brewster_crystal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="button_flatInterface">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/e_flat_interface.png</normaloff>Icons/e_flat_interface.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="button_curvedInterface">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/e_curved_interface.png</normaloff>Icons/e_curved_interface.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="button_thinLens">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/e_thin_lens.png</normaloff>Icons/e_thin_lens.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="button_customElement">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/e_custom_element.png</normaloff>Icons/e_custom_element.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>Icons/t_new.png</normaloff>Icons/t_new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>Icons/t_load.png</normaloff>Icons/t_load.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>Icons/t_save.png</normaloff>Icons/t_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>Icons/t_quit.png</normaloff>Icons/t_quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset>
     <normaloff>Icons/t_edit.png</normaloff>Icons/t_edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Modify cavity</string>
   </property>
  </action>
  <action name="actionCalculator">
   <property name="icon">
    <iconset>
     <normaloff>Icons/t_calcu.png</normaloff>Icons/t_calcu.png</iconset>
   </property>
   <property name="text">
    <string>Calculator</string>
   </property>
   <property name="toolTip">
    <string>Design calculator</string>
   </property>
  </action>
  <action name="view_editElements">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add elements Widget</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="view_calculateSolutions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Calculate solutions Box</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="view_Toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>modifyCavity</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>view_editElements</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>744</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolBar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>view_Toolbar</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
